toolkit/tools/internal/resources/assets/grub2/grub

```diff
@@ -7,7 +7,7 @@ GRUB_CMDLINE_LINUX_DEFAULT="{{.ExtraCommandLine}} $kernelopts"
     
 # =============================notice===============================
 # IMPORTANT: package and feature-specific behaviors are defined in
-#   /etc/default/grub.d/*.cfg. The cfg files are sourced last
+#   /etc/default/grub.d/*.cfg. The executable cfg files are sourced last
```

Comment:
they don't need to be (and shouldn't) be executable...they're intended to be sourced, not run